components/popup/src/vwc-popup-base.ts

```diff
@@ -222,21 +222,23 @@ export class VWCPopupBase extends LitElement {
     }
 
     protected override render(): TemplateResult {
-    	const part = this.alternate ? 'vvd-scheme-alternate' : '';
+    	const alternate = this.alternate ? 'vvd-scheme-alternate' : '';
     	const aria = this.open ? 'false' : 'true';
 
     	return html`
-            <div class="popup-wrapper">
                 <vwc-elevation dp="2">
-                    <div class="popup ${classMap(this.getRenderClasses())}" aria-hidden=${aria} part=${part}>
-                        <div class="popup-content">
+									<!--the popup-wrapper is needed for alternating the inside of the popup nd not its shadow-->
```

Comment:
```suggestion
									<!-- `popup-wrapper` is selected by the wrapping elevation, thus required for shadow styling -->
									<!-- the reason for not applying directly to its first-child `popup` is to avoid scenario where popup is set to alternate scheme and affect the shadow style surfacing contrast which should still in default scheme context -->
```